scripts/dfns-deploy/env-composer.sh

```diff
@@ -16,6 +19,20 @@ yes | cp -f $CONF_FILE_TEMPLATE_DIR_PATH/$SITE_CONF_FILE_NAME $CONF_FILE_DIR_PAT
 yes | cp -f $CONF_FILE_TEMPLATE_DIR_PATH/$SITE_CONF_FILE_NAME $DFNS_AGENT_INSTALL_DIR_PATH/$SITE_CONF_FILE_NAME
 ## Copy dfns conf files
 yes | cp -f $CONF_FILE_TEMPLATE_DIR_PATH/$DFNS_CONF_FILE_NAME $CONF_FILE_DIR_PATH/$DFNS_CONF_FILE_NAME
+yes | cp -f $CONF_FILE_TEMPLATE_DIR_PATH/$DFNS_CLUSTER_CONF_FILE_NAME $CONF_FILE_DIR_PATH/$DFNS_CLUSTER_CONF_FILE_NAME
+yes | cp -f $SHARED_INFO_FILE_PATH/$SHARED_INFO_FILE_NAME $CONF_FILE_DIR_PATH/$SHARED_INFO_FILE_NAME
+
+# set vlan_id_service_url
+DFNS_CLUSTER_PUBLIC_IPS_LIST_PATTERN="dfns_cluster_public_ips_list"
+DFNS_CLUSTER_PUBLIC_IPS_LIST=$(grep $DFNS_CLUSTER_PUBLIC_IPS_LIST_PATTERN $CONF_FILE_DIR_PATH/$DFNS_CLUSTER_CONF_FILE_NAME | awk -F "=" '{print $2}')
+FIRST_DFNS_CLUSTER_IP=${DFNS_CLUSTER_PUBLIC_IPS_LIST%% *}
+
+VLAN_ID_SERVICE_PORT_PATTERN="vlanid_port"
+VLAN_ID_SERVICE_PORT=$(grep $VLAN_ID_SERVICE_PORT_PATTERN $CONF_FILE_DIR_PATH/$SHARED_INFO_FILE_NAME | awk -F "=" '{print $2}')
+
+VLAN_ID_SERVICE_URL=$FIRST_DFNS_CLUSTER_IP:$VLAN_ID_SERVICE_PORT
+
+echo "vlan_id_service_url=$VLAN_ID_SERVICE_URL" >> $CONF_FILE_DIR_PATH/$DFNS_CONF_FILE_NAME
```

Comment:
I think you need to add the protocol here (http://). vlan_id_service_url="http://"$VLAN_ID_SERVICE_URL

Reply:
http or https?

Reply:
http